src/components/App/Test.examples.js

```diff
@@ -0,0 +1,8 @@
+// Test examples
+
+// Test sum function for export
+function sum (i, j) {
+  return i + j
+}
+
+module.exports = sum
```

Comment:
you should use ES6 module loading `export default sum` instead of CommonJS way of module loading `module.exports = sum`